wurst/boss/Minion.wurst

```diff
@@ -0,0 +1,41 @@
+package Minion
+import public EscaperSensor
+import Escaper
+
+public constant COLL_SIZE_MINION = 68.
+
+public class Minion extends EscaperSensorSetup
+    private Escaper target
+
+    construct(vec2 pos, player owner, angle facing)
+        super(pos.toVec3(), owner, getRegionData(pos).getTypeSet().minionId, facing, COLL_SIZE_MINION, COLL_SIZE_MINION + 16)
+        selectTarget()
+
+    construct(vec2 pos, player owner)
+        super(pos.toVec3(), owner, getRegionData(pos).getTypeSet().minionId, COLL_SIZE_MINION, COLL_SIZE_MINION + 16)
+        selectTarget()
+
+    construct(unit existing)
+        super(existing, COLL_SIZE_MINION, COLL_SIZE_MINION + 16)
+        selectTarget()
+
+    override function update()
```

Comment:
I'm not sure if ordering each tick is best performance wise. Maybe just order an attack?